tsconfig-strict.json

```diff
@@ -404,8 +404,6 @@
     "core/templates/pages/exploration-player-page/exploration-player-page.component.spec.ts",
     "core/templates/pages/exploration-player-page/exploration-player-page.constants.ts",
     "core/templates/pages/exploration-player-page/layout-directives/correctness-footer.component.ts",
-    "core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.ts",
```

Comment:
Do we have a issue to track the removal of these files from strict type checks, if yes, can you mention it here ?

Reply:
Will do once the changes are final on this file